test/helpers.bash

```diff
@@ -74,13 +80,15 @@ function run_netavark() {
 ################
 #
 function run_in_container_netns() {
-    run_helper nsenter -n -t $CONTAINER_NS_PID "$@"
+    local i=$1
+    shift 1
+    run_helper nsenter -n -t "${CONTAINER_NS_PIDS[$i]}" "$@"
```

Comment:
same here, 

lets make `0` the default argument so we can remove all the `run_in_container_netns 0` diffs from this PR

Reply:
this one might be weird. The other function is east, check if there is an argument and if so use it but here the number of arguments is unpredictable. Should I for now just say if arg1 == "1" and hardcode it?